web/modules/custom/usa_twig_vars/usa_twig_vars.module

```diff
@@ -132,8 +135,10 @@ function usa_twig_vars_get_page_type(&$node) {
     $term_id = $fields['field_page_type']->getValue()[0]['target_id'];
     $term = Term::load($term_id);
     $page_type = $term ? $term->get('name')->value : '';
-    return $page_type ?? 'content_page';
+    return $page_type ?: 'content_page';
```

Comment:
I want to be sure I understand this one. php.net has either done a slapdash job of documenting `?:` or I didn't find the right page for it. In this case, I think the difference is that the `??` version would have returned '' if `$page_type` is an empty string, and this will return `content_page`. 

... and that we're confident that `$term->get('name')->value` won't return `NULL`.

Reply:
`?:` is shorthand/exploit for the ternary operator and is equivalent to `$page_type ?: $page_type : 'content_page'`. Since the value of `$page_type` won't change if it's set, you can omit it before the `:`.

If `$term->get('name')->value is null`, `$page_type` will be an empty string, so this still returns `$content_page`.